APCAppCore/APCAppCore/Startup/APCAppDelegate.m

```diff
@@ -800,91 +796,87 @@ - (void) clearNSUserDefaults
+    //Profile Tab
+    APCScene *profile = [[APCScene alloc] init];
+    profile.tabBarItem = [[UITabBarItem alloc] initWithTitle:NSLocalizedStringWithDefaultValue(@"Profile", @"APCAppCore", APCBundle(), @"Profile", nil) image:[UIImage imageNamed:@"tab_profile"] selectedImage:[UIImage imageNamed:@"tab_profile_selected"]];
+    profile.tabBarItem.tag = kAPCProfileTabTag;
+    profile.storyboardName = kHealthProfileStoryBoardKey;
+    profile.bundle = [NSBundle appleCoreBundle];
+    [scenes addObject:profile];
     
-    {
-        //Learn Tab
-        UITabBarItem *item = [[UITabBarItem alloc] initWithTitle:NSLocalizedStringWithDefaultValue(@"Learn", @"APCAppCore", APCBundle(), @"Learn", nil) image:[UIImage imageNamed:@"tab_learn"] selectedImage:[UIImage imageNamed:@"tab_learn_selected"]];
-        [tabBarItems addObject:item];
-        
-        UIStoryboard *storyboard = [UIStoryboard storyboardWithName:kLearnStoryBoardKey bundle:[NSBundle appleCoreBundle]];
-        UIViewController *viewController = [storyboard instantiateInitialViewController];
-        [viewControllers addObject:viewController];
-    }
+    return scenes;
+}
+
+- (void)showTabBar
+{
+    self.tabBarController = [[UITabBarController alloc] init];
+    NSUInteger selectedItemIndex = 0;
     
-    {
-        //Profile Tab
-        UITabBarItem *item = [[UITabBarItem alloc] initWithTitle:NSLocalizedStringWithDefaultValue(@"Profile", @"APCAppCore", APCBundle(), @"Profile", nil) image:[UIImage imageNamed:@"tab_profile"] selectedImage:[UIImage imageNamed:@"tab_profile_selected"]];
-        [tabBarItems addObject:item];
-        
-        UIStoryboard *storyboard = [UIStoryboard storyboardWithName:kHealthProfileStoryBoardKey bundle:[NSBundle appleCoreBundle]];
-        UIViewController *viewController = [storyboard instantiateInitialViewController];
-        [viewControllers addObject:viewController];
+    // Set the view controllers
+    NSMutableArray *viewControllers = [NSMutableArray new];
+    for (APCScene *scene in [self tabBarScenes]) {
+        UIViewController *vc = [scene instantiateViewController];
+        [viewControllers addObject:vc];
     }
-    
     [self.tabBarController setViewControllers:[NSArray arrayWithArray:viewControllers]];
-    
-    NSArray *items = self.tabBarController.tabBar.items;
-    
-    for (NSUInteger i=0; i<items.count; i++) {
-        UITabBarItem *item = items[i];
-        UITabBarItem *tabBarItem = tabBarItems[i];
-        
-        item.image = tabBarItem.image;
-        item.selectedImage = tabBarItem.selectedImage;
-        item.title = tabBarItem.title;
-        item.tag = i;
-        
-        if (i == kAPCNewsFeedTabIndex && newsFeedTab){
-            [self updateNewsFeedBadgeCount];
-        }
+
+    // The tab bar icons take the default tint color from UIView Appearance tintin iOS8. In order to fix this for we are selecting each of the tabs.
+    for (NSUInteger ii = 0; ii < self.tabBarController.viewControllers.count; ii++){
```

Comment:
I'm not quite understanding from the comment what issue necessitates this workaround.